src/sources/forus-platform/pug/tpl/directives/blocks/fund-provider-product-editor.pug

```diff
@@ -0,0 +1,113 @@
+.card: form(ng-submit="$dir.form.submit()").form
+    .card-header: .flex.flex-horizontal
+        .flex.flex-grow: .card-title Subsidie starten
+        
+        .flex(ng-if="$dir.deal.active && $dir.onReset"): .block.block-inline-filters
+            button.button.button-text(type="button" ng-click="$dir.onReset({deal: $dir.deal})")
+                .mdi.mdi-trash-can-outline.icon-start
+                | Remove restriction
+
+    .card-section.card-section-primary.form: .row: .col.col-lg-8.col-md-12
+        .form-group.form-group-inline.tooltipped
+            tooltip(text="'Totaal aantal aanbiedingen dat vanuit ' + $dir.fund.name + ' gebruikt mag worden'")
+            .form-label.form-label-required Totaal aantal
+            .form-offset: .row
+                .col.col-lg-6.col-xs-12
+                    input(
+                        required
+                        ng-if="!($dir.product.unlimited_stock && $dir.form.values.limit_total_unlimited)"
+                        ng-model="$dir.form.values.limit_total" 
+                        type="number" 
+                        placeholder="Totaal aanbod" 
+                        min="1" 
+                        max="{{ $dir.product.unlimited_stock ? '' : $dir.product.stock_amount }}").form-control
+                    input(
+                        disabled
+                        ng-if="$dir.product.unlimited_stock && $dir.form.values.limit_total_unlimited"
+                        value='Ongelimiteerd').form-control
+                    .form-error(ng-repeat="error in $dir.form.errors['enable_products.0.limit_total']" ng-bind="error")
+                    .form-limit(ng-if="!$dir.product.unlimited_stock") Limiet 1/{{ $dir.product.stock_amount }}
+                    .form-limit(ng-if="$dir.product.unlimited_stock") Ongelimiteerd
+                
+                .col.col-lg-6.col-xs-12
+                    label(ng-if="$dir.product.unlimited_stock" for='unlimited_stock_subsidy').checkbox
+                        input(type="checkbox" value="1" ng-model="$dir.form.values.limit_total_unlimited" id='unlimited_stock_subsidy')
+                        .checkbox-label 
+                            .checkbox-box: .mdi.mdi-check
+                            | Onbeperkt
+
+        .form-group.form-group-inline.tooltipped
+            tooltip(text="'Hoevaak mag er per inwoner gebruik gemaakt worden van deze aanbieding?'")
+            .form-label.form-label-required Aantal per aanvrager
+            .form-offset: .row
+                .col.col-lg-6.col-xs-12
+                    input(
+                        required
+                        ng-model="$dir.form.values.limit_per_identity" 
+                        type="number" 
+                        placeholder="Limiet per aanvrager" 
```

Comment:
@GerbenBosschieter should we keep it with words per aanvrager or per voucher ( or per aanvrager voucher ), since sponsor can think that it limits somehow by identity, but inside its per voucher